parlai/mturk/core/dev/agents.py

```diff
@@ -298,16 +297,20 @@ def reduce_state(self):
         self.msg_queue = None
         self.recieved_packets = None
 
-    def get_new_act_message(self):
-        """Get a new act message if one exists, return None otherwise"""
+    def check_disconnects(self):
+        """Ensures that an agent is still connected"""
```

Comment:
Kinda weird to have the docstring inverted from the name of the method (check disconnects, check if connected)

Reply:
Maybe assert_connected?